python/huggingfaceserver/huggingfaceserver/__main__.py

```diff
@@ -50,18 +51,22 @@ def list_of_strings(arg):
     _vllm = True
 except ImportError:
     _vllm = False
+
 args, _ = parser.parse_known_args()
 
 if __name__ == "__main__":
-    engine_args = AsyncEngineArgs.from_cli_args(args) if _vllm else None
+    engine_args = None
+    if _vllm and not args.disable_vllm:
+        args.model = args.model_id
+        engine_args = AsyncEngineArgs.from_cli_args(args)
     predictor_config = PredictorConfig(args.predictor_host, args.predictor_protocol,
                                        args.predictor_use_ssl,
                                        args.predictor_request_timeout_seconds)
     model = HuggingfaceModel(args.model_name,
                              predictor_config=predictor_config,
-                             kwargs=vars(args), engine_args=engine_args)
+                             kwargs=vars(args))
     try:
-        model.load()
+        model.load(engine_args=engine_args)
```

Comment:
this changes the load signature, is there a reason why can't be passed to the model instead ?